src/index.html

```diff
@@ -7,6 +7,299 @@
+            <p class="card__cost">$2,199</p>
+          </div>
+
+          <a
+            class="card__button"
+            href="/"
+          >
+            BUY
+          </a>
+        </div>
+
+        <div class="card" >
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="comp"
+          >
+          <p class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+          <p class="card__product">Product code: 195434</p>
+
+          <div class="card__stars stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__reviews">Reviews: 5</div>
+          </div>
+
+          <div class="card__price">
+            <p class="card__text">Price:</p>
+            <p class="card__cost">$2,199</p>
+          </div>
+
+          <a
+            class="card__button"
+            href="/"
+          >
+            BUY
+          </a>
+        </div>
+
+        <div class="card" >
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="comp"
+          >
+          <p class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+          <p class="card__product">Product code: 195434</p>
+
+          <div class="card__stars stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__reviews">Reviews: 5</div>
+          </div>
+
+          <div class="card__price">
+            <p class="card__text">Price:</p>
+            <p class="card__cost">$2,199</p>
+          </div>
+
+          <a
+            class="card__button"
+            href="/"
+          >
+            BUY
+          </a>
+        </div>
+
+        <div class="card" >
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="comp"
+          >
+          <p class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+          <p class="card__product">Product code: 195434</p>
+
+          <div class="card__stars stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__reviews">Reviews: 5</div>
+          </div>
+
+          <div class="card__price">
+            <p class="card__text">Price:</p>
+            <p class="card__cost">$2,199</p>
+          </div>
+
+          <a
+            class="card__button"
+
+            href="/"
```

Comment:
we wrapping our attributes if we have more than 2 or length more than 80
```suggestion
          <a class="card__button" href="/"
```